client/common/cmdline.c

```diff
@@ -243,8 +244,9 @@ BOOL freerdp_client_print_command_line_help_ex(int argc, char** argv,
 	freerdp_client_print_command_line_args(args);
 	printf("\n");
 	printf("Examples:\n");
+	printf("    echo '/v:rdp.freerdp.test' | xfreerdp /from-fd:0\n");
 	printf("    xfreerdp connection.rdp /p:Pwd123! /f\n");
-	printf("    xfreerdp /u:CONTOSO\\JohnDoe /p:Pwd123! /v:rdp.contoso.com\n");
+	printf("    xfreerdp /u:CONTOSO\\JohnDoe /p:Pwd123! /v:rdp.freerdp.test\n");
```

Comment:
Ah ah ah did you get some complains from contoso.com ;)

Reply:
@hardening no, but I thought it might be a good idea to change to something not bothering anyone ^^